xds/internal/balancer/edsbalancer/xds_client_wrapper.go

```diff
@@ -126,50 +123,15 @@ type xdsClientWrapper struct {
 //
 // The given callbacks won't be called until the underlying xds_client is
 // working and sends updates.
-func newXDSClientWrapper(newEDSUpdate func(xdsclient.EndpointsUpdate, error), logger *grpclog.PrefixLogger) *xdsClientWrapper {
+func newXDSClientWrapper(xdsClient xdsClientInterface, newEDSUpdate func(xdsclient.EndpointsUpdate, error), logger *grpclog.PrefixLogger) *xdsClientWrapper {
```

Comment:
Can we get rid of the `xdsClientWrapper` type given that the `xdsClient` is going to be given to this at creation time and does not change afterwards?

Reply:
Ok, let's keep it as it is and revisit at a later time if someone deems it necessary.